src/mondeath.cpp

```diff
@@ -165,6 +165,21 @@ void mdeath::acid(monster *z)
     g->m.add_field(z->pos(), fd_acid, 3, 0);
 }
 
+void mdeath::acidburst(monster *z) // spawn a lot of acid on death
+{
+    if ( g->u.sees(*z) ) {
+        add_msg(m_mixed, _("The %s bursts, showering the nearby area with acid."), z->name().c_str());
```

Comment:
The description is misleading: "showering" implies that it covers it more vertically and could hit head, torso etc., not just at knee level and below.
Also, styling.

Reply:
well, that is indeed a very good point. In that case, it either would need to melt, which then made the whole idea of the Z obsolete and would require it to receive a new name.
Would that be okay ?

Reply:
You could make it spew acidic projectiles like `mdefense::acidsplash`.